compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/symbols/BInvokableSymbol.java

```diff
@@ -56,6 +56,7 @@ public class BInvokableSymbol extends BVarSymbol implements InvokableSymbol {
 
 
     public Set<BVarSymbol> dependentGlobalVars;
+    public HashSet<BInvokableSymbol> childrenFunctions;
```

Comment:
Couldn't find where you are using this field. Also if you are not planning to allow external classes to reassign this shall we make it final

Reply:
It was not used anywhere. Fixed in 75787839cdedf63d7e0fb3f54e5ba1e06282532c.